test/requirements-test.txt

```diff
@@ -15,7 +15,7 @@ git+https://github.com/microsoft/TransformerCompression.git ; python_version >=
 mlflow>=2.4.0
 neural-compressor
 nncf==2.7.0
-nvidia-modelopt~=0.11.0
+nvidia-modelopt[onnx]
```

Comment:
could you add the deps manually? we install specific versions of ort or ort-gpu for the tests. This would override the existing ort we already have installed for the test.

Reply:
sure Jambay .. this test requires cppimport, onnxruntime and onnx-graphsurgeon .. you have idea what is not getting install today in the tests?

Reply:
I think cppimport is the only one missing. is onnxmltools also needed? I see that it is part of the onnx extra but don't know if this example requires it.

Reply:
onnxruntime_extensionsis also required to run this test

Reply:
I added cppimport and onnxmltools to the test requirements file. All other dependencies are already included in the ONNX dependencies within the requirements files. ORT is also required, but since you mentioned it's already being installed, I have not added it.

Reply:
I also made a minor change [here](https://github.com/microsoft/Olive/pull/1441/commits/1dd750b355761f0bcb02e72efb737e56a4d6b7d4#diff-548b5eb07a794e2085e25d700bb5367d06bd7773366b8c9af5d6be4bd80e2501R394). The Windows release version of modelopt (0.17v) only accepts the model path as input, not modelproto. Only Linux release version (0.19) supports modelproto, so I made a change to pass modelpath instead as the olive test install windows 0.17 version